### ISSUE TITLE
Public repo

### ISSUE THREAD

**Issue by cyoung** [STRACTX/eventbrite-get-users#1]
Hey @idohertyr - did you mean to make this repo public? Just checking.

**Comment by idohertyr** [comment]
Yeah, there shouldn't be anything sensitive in this repo and we have limited private repos
